<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerDialog</class>
 <widget class="QDialog" name="ServerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(183, 190, 197);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>会话窗口</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="ServerOutput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>481</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(101, 103, 94);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 127);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ServerInput">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>531</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ServerpushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServerQuit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
